.changeset/tiny-worms-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/client-config': patch
```

Comment:
```suggestion
'@aws-amplify/client-config': minor
```

This looks like feature expansion. Not a fix.

Reply:
Why not a fix? `socialProvider` and `AppClientId` are supposed to be there.

Reply:
Nvm. I don't see `API.md` changing this is file outputs.